Library/Homebrew/cmd/update-bash.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+
+if [ -z "$HOMEBREW_BREW_FILE" ]
+then
+  echo "Error: $(basename "$0") must be called from brew!" >&2
+  exit 1
+fi
+
+brew() {
+  "$HOMEBREW_BREW_FILE" "$@"
+}
+
+which_git() {
+  local which_git
+  which_git="$(which git 2>/dev/null)"
+  if [ -n "$which_git" ] && [ "/usr/bin/git" = "$which_git" ]
+  then
+    local active_developer_dir
+    active_developer_dir="$('/usr/bin/xcode-select' -print-path 2>/dev/null)"
+    if [ -n "$active_developer_dir" ] && [ -x "$active_developer_dir/usr/bin/git" ]
+    then
+      which_git="$active_developer_dir/usr/bin/git"
+    else
+      which_git=""
+    fi
+  fi
+  echo $which_git
+}
+
+git_init_if_necessary() {
+  if [ -d ".git" ]
+  then
+    git init -q
+    git config core.autocrlf false
```

Comment:
Not strictly necessary, but I would prefer to be more explicit about the type of the variable, i.e., `git config --bool core.autocrlf false`.